### DISCUSSION TITLE
How do I set the path for notes in a single org file?

### DISCUSSION THREAD

**Discussion by jlteekiste** [emacs-citar/citar-org-roam#22] [Q&A] [answered]
Hello,

I am migrating my setup for academic work from helm-bibtex to citar and trying to use citar-org-roam to replicate my former setup of saving reference notes under headings in a single org file (inside my org-roam directory).

I'm afraid I can't figure out how to set the path for this file, and to configure citar-org-roam to insert headings in that file rather than adding files to a directory of notes. any pointers would be greatly appreciated.

thanks for your work on citar - it looks like it will make working entirely in emacs feasible for academic work in the humanities - looking forward to getting it all figured out!

**Comment by bdarcus** [comment, marked as answer]
We don't currently have support for creating new notes in a single file, which is why you can't set the path for that. 

The way you would do that now is you'd create the note node, and then use `citar-org-roam-ref-add` to associate it with a reference.

If you want that functionality, you should open a new issue and suggest how it should work. There are a few UX details to sort out.

Worth noting that we have more flexibility. For example, what if a user wants to use multiple files that each hold multiple reference notes? Would there be a variable to set, whose value is a list? And then the UI would prompt you to choose which file to put it in, if more than one?

Also, might some users sometimes want to put a new note in an existing file, and other times in its own file? If yes, how would _that_ work from a UX POV?

PS - if you do create the issue, can you please add the above questions to it?

**Reply by jlteekiste** [reply]
Thanks for the instant reply! Oh I see – I assumed "multiple references per note" and "multiple reference notes per file" might translate to that sort of functionality somehow. Sorry, I'm still trying to figure things out. 

I'll be happy to open an issue once I am a bit more familiar with citar and confident I have understood how things work as it stands. I have just tried out `citar-org-roam-ref-add `and it's certainly a workaround that enables me to continue my work for now.

I'll mull over some of the UX questions you raised – personally, the helm-bibtex solution worked well for me (one large org file for reference notes inside my roam directory, when adding notes to a paper through helm-bibtex, it would automatically insert a new heading with the reference (and I could choose where to refile it), and I had the option of making it an org-roam node by adding an ID.) I'll give it some thought and try to make some clearer suggestions in an issue soon.

**Reply by bdarcus** [reply]
Sounds good.

The current design of citar/citar-org-roam comes out of initial implementation of file-per-note, which is my own bias.

But we had requests for multiple-files-per-note, and I decided I didn't really want to implement that in citar itself. Here we really use org-roam for the note-handling, and so I'd like to do the same for a feature like this, if possible, without hard-coding that.
